src/UncontrolledCollapse.js

```diff
@@ -49,7 +49,7 @@ class UncontrolledCollapse extends Component {
   }
 
   render() {
-    return <Collapse isOpen={this.state.isOpen} {...omit(this.props, omitKeys)} />;
+    return <Collapse isOpen={this.state.isOpen} {...omit(this.props, omitKeys)} aria-expanded={this.state.isOpen ? 'true' : 'false'} />;
```

Comment:
Wouldn't this cascade down to the `Collapse` which would use the value of it's `isOpen` prop similarly to add this?